component/core/src/main/java/org/exoplatform/social/core/jpa/storage/entity/MetadataItemEntity.java

```diff
@@ -141,6 +141,16 @@
         + " AND mi.creatorId = :creatorId"
         + " AND mi.spaceId = :spaceId"
 )
+@NamedQuery(
+    name = "SocMetadataItemEntity.countMetadataItemsByMetadataTypeAndSpacesIdAndCreatorId",
+    query = "SELECT item.spaceId, COUNT(DISTINCT item.id) FROM SocMetadataItemEntity item "
+        + " INNER JOIN SocMetadataEntity sm"
+        + " ON item.metadata.id = sm.id "
+        + " AND sm.type = :metadataType "
+        + " AND sm.audienceId = :creatorId "
+        + " AND item.spaceId IN :spacesId "
```

Comment:
please add parenthesis + plural on last item instead of middle word

```suggestion
        + " AND item.spaceId IN (:spaceIds) "
```